packages/swirl-components/src/components/swirl-chip/swirl-chip.css

```diff
@@ -62,6 +62,15 @@
   }
 }
 
+.chip--variant-plain {
```

Comment:
![image](https://user-images.githubusercontent.com/23700195/224315503-595d8a87-cac9-4dee-bd26-476d48f650b6.png)
issue: the hover effect when a plain variant is selected and interactive is activated looks a little bit squashed as it has no padding. maybe the element should retain the same padding like the outline version, so this does not happen? i don't know the design specs for this...

Reply:
Good catch. The padding wouldn't work well in the tasks card setting. I've added hover styles similar to the plain button.